src/store/actions/todo.js

```diff
@@ -1,4 +1,8 @@
-import { ADD_TASK, DELETE_TASK } from "store/constants/actionTypes";
+import {
+  ADD_TASK,
+  DELETE_TASK,
+  MARK_TASK_DONE,
+} from "store/constants/actionTypes";
 
 export const addTaskToTodo = (task) => {
```

Comment:
As it's an action, we can simply name it addTask and it'll reflect what it's supposed to do.
```suggestion
export const addTask = (task) => {
```